packages/langium/src/service-registry.ts

```diff
@@ -0,0 +1,49 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import { URI, Utils } from 'vscode-uri';
+import { LangiumServices } from './services';
+
+export interface ServiceRegistry {
+    getService(uri: URI): LangiumServices
+    readonly all: LangiumServices[]
+}
+
+export class SingleServiceRegistry implements ServiceRegistry {
```

Comment:
We could provide a function that creates an `ExtensionServiceRegistry` (which could then be named `DefaultServiceRegistry`) for a single language.